packages/charting/configure/src/correct-response.jsx

```diff
@@ -133,9 +133,9 @@ export class CorrectResponse extends React.Component {
     const { categories } = this.state;
     let nextCategories = [];
 
-    if (nextData.length > data.length && nextCorrectAnswerData.length > data.length) {
+    if ((nextData.length > data.length ) || (nextData.length > data.length && nextCorrectAnswerData.length > data.length)) {
       nextCategories = insertCategory(nextCorrectAnswerData, nextData);
```

Comment:
Here you have A || (A && B). Why?

Reply:
hm, I have to check why I had A&& B at the beginning